apps/condo/domains/property/schema/Property.js

```diff
@@ -168,6 +194,56 @@ const Property = new GQLListSchema('Property', {
     },
 })
 
+const PropertyService = new GQLCustomSchema('PropertyService', {
```

Comment:
it's a bad idea to evaluate rank every time in runtime :(
It's better to have some static field that recalculating periodically